dotCMS/src/main/java/com/dotcms/graphql/InterfaceType.java

```diff
@@ -62,15 +48,15 @@
     FORM(FormContentType.class),
     DOTASSET(DotAssetContentType.class);
 
-    private Class<? extends ContentType> baseContentType;
+    final private Class<? extends ContentType> baseContentType;
 
     InterfaceType(final Class<? extends ContentType> baseContentType) {
         this.baseContentType = baseContentType;
     }
 
     public static Set<String> CONTENT_INTERFACE_FIELDS = new HashSet<>();
 
-    private static Map<String, GraphQLInterfaceType> interfaceTypes = new HashMap<>();
+    final private static Map<String, GraphQLInterfaceType> interfaceTypes = new HashMap<>();
```

Comment:
Nitpick: to match every other field, can we declare it in the same order: `private static final...`

Reply:
Done